src/test/scala/nl.knaw.dans.easy.deposit/logging/LoggerSpec.scala

```diff
@@ -47,6 +47,9 @@ class LoggerSpec extends TestSupportFixture with ServletFixture with ScalatraSui
     }
   }
   val stringBuffer = new StringBuilder
+  addServlet(new TestServlet() with CustomResponseLogger with CustomRequestLogger with ResponseLogFormatter with RequestLogFormatter, "/1")
```

Comment:
See https://github.com/janvanmansum/easy-deposit-api/pull/1

Did not yet solve the mystery of broken tests.

Reply:
Lessons learnt with 54b6c29 of the above accumulate pull request

* We should not use `header("???")` shouldBe "???",
  it would not detect multiple values for one key.
  This means scanning and fixing all servlet tests in all projects.
* AuthServlet should not refresh cookies, only DepositServlet should.
  This one is a challenge to fix.